build-script.sh

```diff
@@ -4,12 +4,24 @@
 TMPFILE=`mktemp /tmp/SwiftUIcon.swift.XXXXXX` || exit 1
 trap "rm -f $TMPFILE" EXIT
 
-if [ -z "$SCRIPT_INPUT_FILE_0" ]
-then
+
+[[ -s "$SCRIPT_INPUT_FILE_0" ]] || {
     echo "error: You must specify your Icon.swift as the first Input File in the Build Phase."
     exit 1
-fi
+}
+
+[[ -s "$SCRIPT_OUTPUT_FILE_0" ]] || {
```

Comment:
Thanks for adding the Assets.xcassets as an output. I think that makes a lot of sense 👍